test/unit/ngsiv2/httpBindings-test.js

```diff
@@ -786,4 +786,270 @@ describe('HTTP Transport binding: measures', function() {
             });
         });
     });
-});
+
+    describe('When a POST measure arrives with a TimeInstant attribute in the body for a device with explicitAttrs:false', function() {
+        var timeInstantRequest = {
+                url: 'http://localhost:' + config.http.port + '/iot/d',
+                method: 'POST',
+                qs: {
+                    i: 'timestampedDevice',
+                    k: '1234'
+                },
+                headers: {
+                    'Content-type': 'text/plain'
+                },
+                body: 'tmp|24.4|humidity|32|tt|2020-06-14T12:19:26.476659Z'
+            },
+            provisionProduction = {
+                url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
+                method: 'POST',
+                json: utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json'),
+                headers: {
+                    'fiware-service': 'smartGondor',
+                    'fiware-servicepath': '/gardens'
+                }
+            };
+
+        beforeEach(function(done) {
+            // This mock does not check the payload since the aim of the test is not to verify
+            // device provisioning functionality. Appropriate verification is done in tests under
+            // provisioning folder of iotagent-node-lib
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities?options=upsert')
+                .reply(204)
+                .post(
+                    '/v2/entities/TimeInstant%20New%20Device/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')                )
+                .query({ type: 'clock' })
```

Comment:
We have:

* First test case:
    * L806: `utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json')`
    * L824: `utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')`
* Second test case:
    * L872: `utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant2.json')`
    * L890: `utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json')`
* Third test case:
    * L938: utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json')
    * L956: ` utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')`
* Fourth test case:
    * L1004: `utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant2.json')`
    * L1022: `utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json')`

Could some of the .json files be renamed so we have aligned pairs in "2" and "no-2":

```
utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json')
utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json')
```

and

```
utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant2.json')
utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')
```

Reply:
> We have:
> 
> * First test case:
>   
>   * L806: `utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json')`
>   * L824: `utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')`
> * Second test case:
>   
>   * L872: `utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant2.json')`
>   * L890: `utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json')`
> * Third test case:
>   
>   * L938: utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json')
>   * L956: ` utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')`
> * Fourth test case:
>   
>   * L1004: `utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant2.json')`
>   * L1022: `utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json')`
> 
> Could some of the .json files be renamed so we have aligned pairs in "2" and "no-2":
> 
> ```
> utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant.json')
> utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures.json')
> ```
> 
> and
> 
> ```
> utils.readExampleFile('./test/unit/ngsiv2/deviceProvisioning/provisionDeviceTimeinstant2.json')
> utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')
> ```

@fgalan Hi, I have been incorporated the changes suggested by you. Thanks!